tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -2014,10 +1639,52 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         jqUnit.assertNoValue("The change request has destroyed the child component", that.child);
     });
     
+    fluid.defaults("fluid.tests.fluid5632root1", {
+        gradeNames: ["fluid.modelComponent"],
+        model: {
+            value1: NaN,
+            value2: "{that}.model.value1"
+        }
+    });
+    
+    fluid.tests.isNaN = function (value) { // replicate for Number.isNaN which is not supported on IE
```

Comment:
Minor detail: Number.isNaN is also [not supported on Safari](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN#Browser_compatibility).